plugins/webp-uploads/picture-element.php

```diff
@@ -98,6 +98,11 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 	// Get the sizes from the IMG tag.
 	$sizes = $processor->get_attribute( 'sizes' );
 
+	// Bail early when the responsive images are disabled.
+	if ( null === $sizes ) {
+		return $image;
+	}
```

Comment:
While it's not recommended, it's possible to have a `srcset` attribute on an image without a `sizes` attribute, in which case the browser will use the default value of `100vw` for `sizes`. We should probably check for the existence of both a `srcset` and `sizes` attribute, which is what WordPress also checks for before modifying the img tag ([here](https://github.com/WordPress/wordpress-develop/blob/6.6/src/wp-includes/media.php#L1778)).

Reply:
Thanks. Removed in https://github.com/WordPress/performance/pull/1449/commits/7e09a7231dc04ef3772c0be58fae185dc89a3b7b